### PR TITLE
Common config files that should not be spell checked

### PR DESCRIPTION
**Pull request by bart1**
In R projects/packages I commonly have a gitlab ci/cd and pkgdown configuration file. I think these should not be spell checked therefore these suggestions for the defaults of the config files

### CODE REVIEW

**Comment by bart1** [comment]
I now realize `.lintr` might be an other file to consider

**Comment by lorenzwalthert** [comment]
Hi @bart1, thanks for your suggestions. I agree in the lintr spec file, but in my experience, pkgdown should contain very few special words apart from plain English and the names of the functions in your package. the text specified in that file is also user visible to a large extend, so I think we best don’t exclude it. The gitlab ci i am happy to add to the template. Note though that the listed refer are alphabetically ordered. Please adjust that. 

Also, note that you can override the config in this repo in your own `.pre-commit-config.yaml`.

**Comment by bart1** [comment]
@lorenzwalthert Thank you for your suggestions, I have updated the merge request accordingly.
ps I do indeed make these changes in my own `.pre-commit-config.yaml` I was however I was making the same changes in multiple repositories and thought these might be better defaults

File: inst/pre-commit-config-pkg.yaml [hunk 1]
```diff
@@ -25,13 +25,13 @@ repos:
           .*\.Rproj|
           .*\.sh|
           (.*/|)\.gitignore|
+          (.*/|)\.gitlab-ci.yml|
```
Comment (lorenzwalthert) [inline inst/pre-commit-config-pkg.yaml:28]:
```suggestion
          (.*/|)\.gitlab-ci\.yml|
```

File: inst/pre-commit-config-proj.yaml [hunk 2]
```diff
@@ -21,6 +21,8 @@ repos:
           .*\.Rproj|
           .*\.sh|
           (.*/|)\.gitignore|
+          (.*/|)\.gitlab-ci.yml|
```
Comment (lorenzwalthert) [inline inst/pre-commit-config-proj.yaml:24]:
```suggestion
          (.*/|)\.gitlab-ci\.yml|
```

**Comment by lorenzwalthert** [comment]
@bart1 I see. You can also set an R option to use a local or remote config template, see argument `config_source` in `precommit::use_precommit()`.

**Pull request merged**